unix/snowagent.config

```diff
@@ -110,13 +120,25 @@
         </Include>
     </Oracle>
     <Logging>
-        <MaxSize>10240</MaxSize>
+        <MaxSize>2048</MaxSize>
         <!-- error, warning, info, trace, verbose -->
-        <Level>verbose</Level>
+        <Level>warning</Level>
     </Logging>
     <SystemSettings>
         <Setting key="log.append" value="true" />
+		<!-- Scan jar files -->
+		<Setting key="software.scan.jar" value="true"/>
+		<!-- Standard file system on AIX -->
```

Comment:
Those are active via the default config already. Why have them here?

Reply:
again - why? do we want to motivate people do disable it? Or is the default not working, what then should be handled elsewhere?

Reply:
removed the setting